crosscompilation fails for 0.14.68

compiler: mips-openwrt-linux-musl-g++
works with 0.14.67
fails with 0.14.68
output:

tcptunserver.cpp: In member function 'virtual void TcpTunConn::stop(bool)':
tcptunserver.cpp:204:3: error: 'close' was not declared in this scope; did you mean 'pclose'?
  204 |   close(fd);
      |   ^~~~~
      |   pclose
tcptunserver.cpp: In destructor 'virtual TcpTunServerBase::~TcpTunServerBase()':
tcptunserver.cpp:507:5: error: 'close' was not declared in this scope; did you mean 'pclose'?
  507 |     close(fd);
      |     ^~~~~
      |     pclose
tcptunserver.cpp: In member function 'virtual void TcpTunServerBase::stop(bool)':
tcptunserver.cpp:530:7: error: 'close' was not declared in this scope; did you mean 'pclose'?
  530 |       close(fd);
      |       ^~~~~
      |       pclose
tcptunserver.cpp: In member function 'virtual void TcpTunServer::start()':
tcptunserver.cpp:607:3: error: 'close' was not declared in this scope; did you mean 'pclose'?
  607 |   close(fd);
      |   ^~~~~
      |   pclose
tcptunserver.cpp: In member function 'virtual void UnixTunServer::start()':
tcptunserver.cpp:680:21: error: '::unlink' has not been declared; did you mean 'inline'?
  680 |               if (::unlink(path.c_str()) == -1 && errno != ENOENT)
      |                     ^~~~~~
      |                     inline
tcptunserver.cpp:710:3: error: 'close' was not declared in this scope; did you mean 'pclose'?
  710 |   close(fd);
      |   ^~~~~
      |   pclose
tcptunserver.cpp: In destructor 'virtual TcpTunSystemdServer::~TcpTunSystemdServer()':
tcptunserver.cpp:772:7: error: 'close' was not declared in this scope; did you mean 'pclose'?
  772 |       close(fd);
      |       ^~~~~
      |       pclose

--- a/src/libserver/tcptunserver.cpp
+++ b/src/libserver/tcptunserver.cpp
@@ -21,6 +21,7 @@
 #include "config.h"
 #include "tunchannel.h"
 
+#include <unistd.h>
 #include <netinet/tcp.h>
 #include <sys/un.h>
 #include <arpa/inet.h>

